include/model/CIndividualModel.h

```diff
@@ -25,6 +25,8 @@
 #include <utility>
 #include <vector>
 
+class CTestHelper;
```

Comment:
I don't think we should have something this specific in the global namespace called `CTestHelper`. I would move this into `ml::model` and probably call it `CIndividualModelTestHelper`.

Reply:
Renamed